java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -0,0 +1,1359 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.bond;
+
+import com.microsoft.bond.protocol.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Partially implements the {@link BondType} contract for struct data types.
+ * Leaves some implementation details to subclasses specific to the a particular
```

Comment:
I changed "some" to "the rest". If you want to know what they are, look at the abstract methods.

---
In reply to: [123382905](https://github.com/Microsoft/bond/pull/502#discussion_r123382905) [](ancestors = 123382905)